test/membrane_rtc_engine/webrtc/track_sender_test.exs

```diff
@@ -150,6 +162,13 @@ defmodule Membrane.RTC.Engine.WebRTC.TrackSenderTest do
     Pipeline.terminate(pipeline, blocking?: true)
   end
 
+  defp build_h264_track() do
+    Track.new(:video, @stream_id, @track_origin, :H264, 90_000, :raw, nil,
+      id: @track_id,
+      simulcast_encodings: @simulcast_encodings
+    )
+  end
```

Comment:
Yeah, I tried to do this but both files use attributes like `@simulcast_encodings`. I would have to make functions for them and call them like 

```elixir
Enum.each(Utils.simulcast_encodings, fn en -> assert ... end)

# instead of 

Enum.each(@simulcast_encodings, fn en -> assert ... end)
```